packages/upswyng-web/src/components/Temperature.tsx

```diff
@@ -36,42 +34,20 @@ const StyledLocation = styled.span`
   padding: 0;
 `;
 
-const StyledTooltip = styled(props => (
-  <Tooltip
-    classes={{ popper: props.className, tooltip: "tooltip" }}
-    {...props}
-  />
-))`
-  & .tooltip {
-    background: ${colors.black};
-    colors: ${colors.white};
-    font-family: ${font.families.openSans};
-  }
-` as typeof Tooltip;
-
 const Temperature = () => {
   const temp: undefined | null | number = useTemperature();
   return (
     <StyledContainer>
       <Typography variant="srOnly" component="h2">
         Current Temperature
       </Typography>
-      <StyledTemp>
-        {temp === undefined && (
-          <Skeleton variant="rect" height={19} width={40} />
-        )}
-        {temp === null && (
-          <StyledTooltip
-            placement="left"
-            title="temperature currently unavailable"
-          >
-            <ErrorOutlineIcon fontSize="small" tabIndex={0} />
-          </StyledTooltip>
-        )}
-        {temp && <StyledDegrees>{temp}&deg;</StyledDegrees>}
-        <Typography variant="srOnly">in</Typography>
-        <StyledLocation>Boulder, CO</StyledLocation>
-      </StyledTemp>
+      {temp && (
```

Comment:
I think this should be applied to the entire component rather than just the temperature. At the top of this component is a "Current Temperature" heading that is visible to only screen readers and could be confusing when the temperature fails to load.